src/data/interfaces/run.interfaces.ts

```diff
@@ -0,0 +1,32 @@
+import { Date, LocalizedString, Teams } from './'
+
+export enum EventType {
+  Start,
+  Finish,
+  ChallengeCompleted,
+}
+
+export enum RunGameState {
+  Planned,
+  Running,
+  Finished,
+}
+
+export interface Event {
+  type: EventType
+  timestamp: Date
+  team?: string
+  challenge?: string
```

Comment:
Perhaps even the challenge could be an enum? 🤔 It's probably gonna be much easier to create the events if it was an enum 😅  But it's not needed for this ticket. We can do it in the ticket about admin DX we created yesterday :)

Reply:
🤔 indeed! it will make easier to maintain the events data. the only point i see is that we are, kind of, putting inside the code details of the data. 
for instance: on a first event we have `team-a`, `team-b`, if on a second event we have a `team-c` we need to change the code, not only the data. it's a very simple change, of course, but we need to change the code to attend some change on the data.

but let's discuss it better on the DX ticket, for sure!

btw, to make it safer, i created this validation on the engine. at least if the data is wrong, the code will fail fast and we'll be able to quickly fix it
https://github.com/geo-quest/travel-bingo/blob/f374cbeba81227a4518a1e5bfd11eda7223ebd88/src/engine/calculate-score.ts#L132-L136

Reply:
Yeah, it makes sense. let's discuss it on the DX ticket ;) 

And love the validation 😍